sql_user_check_snowflake/run.sh

```diff
@@ -8,13 +8,28 @@ readonly USERNAME=$SF_USERNAME
 readonly DATABASE=$SF_DATABASE
 readonly WAREHOUSE=$WAREHOUSE_NAME
 readonly ROLE=$ROLE_NAME
+readonly MODE=$MODE
+readonly OUTPUT=$OUTPUT
 
 if ! command -v "$CMD" &> /dev/null
 then
     echo "Command $CMD could not be found, Follow https://docs.snowflake.com/en/user-guide/snowsql-install-config to install $CMD"
     exit 1
 fi
 
-tempQueryFile=$(mktemp)
-sed "s/LB_DATABASE_NAME/$DATABASE/g" queries.sql > "$tempQueryFile"
-snowsql -a "$ACCOUNT" --username "$USERNAME" -f "$tempQueryFile" -w "$WAREHOUSE" -r "$ROLE"
+
+if [ "$MODE" == "stats" ]; then
+  tempQueryFile=$(mktemp)
+  sed "s/LB_DATABASE_NAME/$DATABASE/g" stats_queries.sql > "$tempQueryFile"
+  snowsql -a "$ACCOUNT" --username "$USERNAME" -f "$tempQueryFile" -w "$WAREHOUSE" -r "$ROLE" -o output_file="$OUTPUT"
+
+elif [ "$MODE" == "full_metadata" ]; then
+  tempQueryFile=$(mktemp)
+  sed "s/LB_DATABASE_NAME/$DATABASE/g" queries.sql > "$tempQueryFile"
```

Comment:
queries.sql is not there in sql_user_check_snowflake folder

Reply:
Its there, its not part of this PR since i haven't changed anything there.